packages/zowe-explorer-api/src/fs/types/jobs.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ *
+ */
+
+import { IJob, IJobFile } from "@zowe/zos-jobs-for-zowe-sdk";
+import { DirEntry, EntryMetadata, FileEntry } from "../types";
+import { FilePermission, FileType } from "vscode";
+
+export type JobFilter = {
+    searchId: string;
+    owner: string;
+    prefix: string;
+    status: string;
+};
+
+export class SpoolEntry extends FileEntry {
+    public name: string;
+    public type: FileType;
+    public wasAccessed: boolean;
```

Comment:
The comment about duplication/re-defining properties may apply to various classes across the board